pkgs/data/fonts/oxygenfonts/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub }:
+
+let
+  pname = "oxygenfonts";
+  version = "unstable-20140213";
+in
+
+  stdenv.mkDerivation rec {
+    name = "${pname}-${version}";
+
+    src = fetchFromGitHub {
+      owner = "vernnobile";
+      repo = "oxygenFont";
+      rev = "62db0ebe3488c936406685485071a54e3d18473b";
+      sha256 = "134kx3d0g3zdkw8kl8p6j37fzw3bl163jv2dx4dk1451f3ramcnh";
+    };
+
+    phases = [ "unpackPhase" "installPhase" ];
+
+    installPhase = ''
+      mkdir -p $out/share/fonts/${pname}
+      cp */*.ttf $out/share/fonts/${pname}
```

Comment:
That was the intention, but I reread the README and found that the Google fonts were the simplified version. So I changed it to take the latest (0.4) full release, plus the Mono font.